mathesar/views.py

```diff
@@ -107,14 +107,15 @@ def render_schema(request, database, schema):
 
 def home(request):
     database = get_current_database(request, None)
-    schema = get_current_schema(request, None, database)
-    return render_schema(request, database, schema)
+    return redirect('schemas', db_name=database.name)
 
 
-def db_home(request, db_name):
-    database = get_current_database(request, db_name)
-    schema = get_current_schema(request, None, database)
-    return render_schema(request, database, schema)
+# There is no db_home.
+# db_home and schemas are the same page in FE
+# def db_home(request, db_name):
+#     database = get_current_database(request, db_name)
+#     schema = get_current_schema(request, None, database)
+#     return render_schema(request, database, schema)
```

Comment:
These comments can be removed.

Reply:
agreed.

Reply:
@rajatvijay I've added https://github.com/centerofci/mathesar/pull/1634/commits/4efd8f214c7fe835e39c9d3e463591cebd574b08 to resolve this.